### PR TITLE
Fix back in time button and adds 2d meshes

### PR DESCRIPTION
**Pull request by naomijub**
- Button `|<<` had to be removed, the functionality is no longer available
- Clears Inspector and Hierarchy Filters
- Shapes are now spawnable and visible (Size was too small)
- Fixed some scenes, but scenes hierarchies seem to be incorrect due to new hashing loginc

### CODE REVIEW

**Comment by rewin123** [comment]
I looked at the PR. Looks great, and I understand some of the scenes have been fixed, which is pretty cool too. I don't like the only change I noted in the review

**Comment by naomijub** [comment]
> I looked at the PR. Looks great, and I understand some of the scenes have been fixed, which is pretty cool too. I don't like the only change I noted in the review

SOrry, I didn't get which note you left in the review. I dont see it

**Review by rewin123 — CHANGES_REQUESTED** [review]

**Comment by rewin123** [comment]
Oh, I didn't realise I had to hit the complete review button. Can you see it now?

File: crates/prefab/src/plugins.rs [hunk 1]
```diff
@@ -73,18 +75,21 @@ impl Plugin for BasePrefabPlugin {
...
         app.editor_registry::<GltfPrefab>();
+        app.editor_relation::<GltfPrefab, PrefabMarker>();
         app.editor_registry::<MaterialPrefab>();
         app.editor_registry::<ColorMaterialPrefab>();
 
         app.editor_registry::<Sprite>();
         app.editor_registry::<SpriteTexture>();
         app.editor_relation::<SpriteTexture, Transform>();
         app.editor_relation::<SpriteTexture, Visibility>();
+        app.editor_relation::<Sprite, PrefabMarker>();
```
[the remark this answers is not in the document]
Comment (naomijub) [reply crates/prefab/src/plugins.rs:86]: That makes sense, however, if we create a scene only using the bundle components, the saved scene would not work well. What can I do is determine if it is a root object, and if it is we add a prefab marker. 

So I can create a new sync function that removes prefab marker if the entity has parent

**Pull request merged**